src/services/AuthorizationRequestProcessor.ts

```diff
@@ -77,12 +78,23 @@ export class AuthorizationRequestProcessor {
 					state: session?.state,
 				};
 
+				try {
+					await this.f2fService.sendToTXMA({
+						event_name: "F2F_CRI_END",
+						...buildCoreEventFields(session, this.environmentVariables.issuer(), session.clientIpAddress, absoluteTimeNow),
+
+					});
+				} catch (error) {
+					this.logger.error("Failed to write TXMA event F2F_CRI_END to SQS queue.");
```

Comment:
Please can we have a messageCode for this error, and maybe it would be helpful to also include the error object? I'm thinking something like 

`this.logger.error("Failed to write TXMA event F2F_CRI_END to SQS queue", { error, messageCode: MessageCodes.XXXX });`

To ensure a consist structure of error logs